### ISSUE TITLE
nxos_config replace config does not replace existing configuration

### ISSUE THREAD

**Issue by rvitasek** [ansible/ansible#60417]
##### SUMMARY
Replace config does not remove existing configuration. It only executes commands included in the replace_src file. Therefore, the commands are merged into existing config on the switch rather than replaced.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
nxos_config
replace: config

##### ANSIBLE VERSION

```
ansible 2.8.2
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/admin/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Jun 20 2019, 20:27:34) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

##### CONFIGURATION

Default - no change.

##### OS / ENVIRONMENT
Nexus 9000 / 7.0(3)I7(3) / 7.0(3)I7(6) / 9.2(3)

##### STEPS TO REPRODUCE

replace.yml
```
    - name: Replace configuration
      nxos_config:
        replace_src: "{{ lookup('file', '~/config/default.cfg') }}"
        replace: config
```

ansible-playbook replace.yml -vvvv
```
    "updates": [
        "config replace ", 
        "!Command: show running-config", 
        "!Time: Mon Aug 12 02:46:26 2019", 
        "", 
        "version 7.0(3)I7(3)", 
        "hostname N9K-1", 
        "vdc N9K-1 id 1", 
```

The first command seen in the accounting log is 'configure terminal ; version 7.0(3)I7(3)':

N9K-1# show accounting log
```
Mon Aug 12 13:03:17 2019:type=stop:id=x.x.x.x@pts/2:user=admin:cmd=shell terminated because the ssh session closed
Mon Aug 12 13:11:16 2019:type=start:id=x.x.x.x@pts/2:user=admin:cmd=
Mon Aug 12 13:11:16 2019:type=update:id=x.x.x.x@pts/2:user=admin:cmd=terminal length 0 (SUCCESS)
Mon Aug 12 13:11:16 2019:type=update:id=x.x.x.x@pts/2:user=admin:cmd=terminal width 511 (SUCCESS)
Mon Aug 12 13:11:20 2019:type=update:id=x.x.x.x@pts/2:user=admin:cmd=configure terminal ; version 7.0(3)I7(3) (SUCCESS)
Mon Aug 12 13:11:20 2019:type=update:id=x.x.x.x@pts/2:user=admin:cmd=configure terminal ; hostname N9K-1 (SUCCESS)
Mon Aug 12 13:11:20 2019:type=update:id=x.x.x.x@pts/2:user=admin:cmd=configure terminal ; vdc N9K-1 id 1 (SUCCESS)
```

This is because 'config replace ' command without path to a file is not a valid command:

```
N9K-1# config replace 
                                  ^
% Incomplete command at '^' marker.
```
```
N9K-1# config replace ?
  bootflash:  Configuration file to use
  commit      Commit config replace operation
  scp:        Configuration file to use
  sftp:       Configuration file to use
```

##### EXPECTED RESULTS

The running configuration should be replaced with the provided file, not merged.

Here is an example of config replace on the switch:

```
N9K-1# config replace bootflash:default.cfg 

Version match between user file and running configuration.
Pre-check for User config PASSED
ADVISORY: Config Replace operation started...
Modifying running configuration from another VSH terminal in parallel
is not recommended, as this may lead to Config Replace failure.

Collecting Running-Config
Generating Rollback patch for switch profile
Rollback Patch is Empty
Collecting Running-Config
Generating Rollback Patch
Executing Rollback Patch 
During CR operation,will retain L3 configuration
when vrf member change on interface
Generating Running-config for verification
Generating Rollback Patch

Configure replace completed successfully. Please run 'show config-replace log exec' to see if there is any configuration that requires reload to take effect.
```

##### ACTUAL RESULTS

Same as above.

**Comment by ansibot** [comment]
Files identified in the description:
* [`lib/ansible/modules/files/replace.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/files/replace.py)
* [`lib/ansible/modules/network/nxos/nxos_config.py`](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_config.py)

If these files are inaccurate, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @EvanK @chrisvanheuveln @mikewiebe @privateip @rahushen @rcarrillocruz @trishnaguha
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@rvitasek, just so you are aware we have a dedicated Working Group for network.
You can find other people interested in this in `#ansible-network` on Freenode IRC
For more information about communities, meetings and agendas see https://github.com/ansible/community

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
cc @EvanK @chrisvanheuveln @mikewiebe @privateip @rahushen @rcarrillocruz @trishnaguha
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by rvitasek** [comment]
My bad, the file needs to be local on remote switch.
